docs/podman.1.md

```diff
@@ -23,6 +23,9 @@ has the capability to debug pods/images created by crio.
 **--help, -h**
   Print usage statement
 
+**--cgroup-manager**
```

Comment:
A blog showing how this works would probably be nice once this hits the streets.